src/main/app/controller/GetController.ts

```diff
@@ -7,23 +7,28 @@ import { AppRequest } from './AppRequest';
 
 @autobind
 export class GetController {
-  constructor(protected readonly name: string, protected readonly content: Record<string, Record<string, unknown>>) {}
+  constructor(
+    protected readonly view: string,
+    protected readonly content: Record<'en' | 'cy' | 'common', Record<'divorce' | 'civil' | string, unknown>>
```

Comment:
Create an alias for that type below

Reply:
👍 Good idea, I've added a new type for this in 9b52937.